Header header

float64 throttle
uint32 mode

float64 priority
float64 lifetime